test/src/ott-analytics.spec.js

```diff
@@ -314,6 +315,24 @@ describe('_sendAnalytics', () => {
     spy.restore();
   });
 
+  it('should stop sending requests after destroy', done => {
+    let numberOfCalls;
+    const checkStopWorking = () => {
+      spy.callCount.should.equal(numberOfCalls);
```

Comment:
add `try{...}catch(e){done(e)}` to catch errors in mocha async tests otherwise they will get muted.
Also, I think you need to check the internal `_mediaHitInterval` to be null
and also check it for both reset and destroy, so add another test

Reply:
Wired, I saw others tests without try catch, i'll check it and add it all tests if needed.
I didn't check `_mediaHitInterval ` because I wanted to do generic test.

Reply:
So it just means they are written wring cause id they fail the exception is not caught as test fail, but as timeout, see [here](https://stackoverflow.com/questions/28181015/assert-is-breaking-async-function-in-mocha-test) for example.